src/PrestaShopBundle/Service/Tab/TabStatus.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace PrestaShopBundle\Service\Tab;
+
+use Doctrine\ORM\EntityManagerInterface;
+use PrestaShopBundle\Entity\Repository\TabRepository;
+use PrestaShopBundle\Entity\Tab;
+
+class TabStatus
```

Comment:
class name make me think it's an entity or an object model... how about declare a TabRepository if it doesn't exist and implement changeStatusByClassName inside it? The implementation of the function is OK :+1: